### PR TITLE
Fabo/move to GitHub actions

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
nice! why?

**Comment by faboweb** [comment]
To allow running e2e tests just on pull requests. Currently, we need to run on all the commits because the releases run on pushes to the release branch. The changelog check is just a bonus.

File: .github/workflows/checkChangelog.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+      # What you should use
+      - name: Add simsala
+        run: npm install simsala
+      - name: Check if changlog was updated
+        run: node node_modules/simsala/src/cli.js check --pending-path ./changes
```
Comment (Bitcoinera) [inline .github/workflows/checkChangelog.yml:19]: Where is it triggering here CircleCI? Or are they two separate things? Getting a bit confused here...

Comment (faboweb) [reply .github/workflows/checkChangelog.yml:19]: this is using GithubActions

Comment (faboweb) [reply .github/workflows/checkChangelog.yml:19]: https://github.com/luniehq/lunie/pull/3859/checks?check_run_id=575492006

**Pull request merged**